test/functional/normal/tabpage_spec.lua

```diff
@@ -0,0 +1,38 @@
+local helpers = require('test.functional.helpers')(after_each)
+
+local clear = helpers.clear
+local command = helpers.command
+local eq = helpers.eq
+local feed = helpers.feed
+local eval = helpers.eval
+
+describe('tabpage', function()
+  before_each(clear)
+
+  it('advances to the next page via <C-W>gt', function()
```

Comment:
This test is not running on a terminal buffer unlike the test in `test_terminal.vim`. Same issue with the other test in this file.